go.mod

```diff
@@ -25,12 +25,12 @@ require (
 	github.com/lightninglabs/lightning-node-connect/hashmailrpc v1.0.2
 	github.com/lightninglabs/lndclient v0.16.0-15
 	github.com/lightninglabs/neutrino/cache v1.1.1
-	github.com/lightningnetwork/lnd v0.17.0-beta.rc2
-	github.com/lightningnetwork/lnd/cert v1.2.1
-	github.com/lightningnetwork/lnd/clock v1.1.0
-	github.com/lightningnetwork/lnd/ticker v1.1.0
-	github.com/lightningnetwork/lnd/tlv v1.1.0
-	github.com/lightningnetwork/lnd/tor v1.1.1
+	github.com/lightningnetwork/lnd v0.17.0-beta.rc5
```

Comment:
Can we also bump `aperture` to `v0.2-beta` (will resolve to `v0.1.21-beta.0.20230705004936-87bb996a4030` for some reason, maybe because we didn't follow semver versioning?) and `lndclient` to `v0.17.0-0` please?

Reply:
> maybe because we didn't follow semver versioning?)

Yeah I think it needs an extra 0 at the end there in the tag? so v0.2.0-beta.

Reply:
I think so, yes. No idea what other reason there could be for go modules turning a proper tag into a pseudo version...